cpp/src/io/json/nested_json_gpu.cu

```diff
@@ -843,34 +1008,30 @@ void get_token_stream(device_span<SymbolT const> json_in,
  * first node encountered in \p input
  * @param[in] input The JSON input in host memory
  * @param[in] d_input The JSON input in device memory
+ * @param[in] options Parsing options specifying the parsing behaviour
  * @param[in] stream The CUDA stream to which kernels are dispatched
+ * @param[in] mr Optional, resource with which to allocate
```

Comment:
Similar to above, remove `[in]`.

Reply:
Since we do have both `[in]` and `[in,out]` params, I would prefer to be rather over-communicating and keep the `in`/`out` specs in this specific case.